src/golang/cmd/server/handler/get_artifact_result.go

```diff
@@ -40,16 +47,22 @@ type getArtifactResultArgs struct {
 	artifactId          uuid.UUID
 }
 
-type getArtifactResultResponse struct {
+type artifactResultMetadata struct {
 	Name string `json:"name"`
 
 	// `Status` is redundant due to `ExecState`. Avoid consuming `Status` in new code.
 	// We are incurring this tech debt right now since there are quite a few usages of
 	// `status` in the UI.
-	Status    shared.ExecutionStatus `json:"status"`
-	ExecState shared.ExecutionState  `json:"exec_state"`
-	Schema    []map[string]string    `json:"schema"`
-	Data      string                 `json:"data"`
+	Status            shared.ExecutionStatus            `json:"status"`
```

Comment:
I'm currently doing some work on the UI for showing Artifact and Metric details in their own pages.

Perhaps we can take on this status refactor as part of my work as well?

Reply:
https://linear.app/aqueducthq/issue/ENG-1591/refactor-use-execstate-instead-of-status-on-ui